scripts/bootstrap-aio.yml

```diff
@@ -151,12 +151,15 @@
     rpco_deploy_hardening: "{{ lookup('env', 'DEPLOY_HARDENING') }}"
     TRIGGER: "{{ lookup('env', 'TRIGGER')}}"
     SCENARIO: "{{ lookup('env', 'SCENARIO')}}"
-    TARGET: "{{ lookup('env', 'TARGET')}}"
+    TARGET: "{{ target_override | default(lookup('env', 'TARGET')) }}"
```

Comment:
I have it in the job, but I also added exporting TARGET in the configure-multi-node-aio.sh so that the script can easily be run outside of the gate.